recipe/meta.yaml

```diff
@@ -10,6 +10,7 @@ source:
   git_depth: 60
 
 build:
+  noarch: python
   number: 0
```

Comment:
Also would recommend bumping the build number.

Reply:
Sure!